gui-apps/waybar/waybar-0.9.17-r1.ebuild

```diff
@@ -0,0 +1,98 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit meson
+
+DESCRIPTION="Highly customizable Wayland bar for Sway and Wlroots based compositors"
+HOMEPAGE="https://github.com/Alexays/Waybar"
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/Alexays/${PN^}.git"
+else
+	SRC_URI="https://github.com/Alexays/${PN^}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64"
+	S="${WORKDIR}/${PN^}-${PV}"
+fi
+
+LICENSE="MIT"
+SLOT="0"
+IUSE="evdev experimental jack libinput logind mpd mpris network +popups pipewire pulseaudio sndio systemd test tray +udev upower wifi"
+
+RESTRICT="!test? ( test )"
+
+BDEPEND="
+	>=app-text/scdoc-1.9.2
+	dev-util/gdbus-codegen
+	virtual/pkgconfig
+"
+RDEPEND="
+	dev-cpp/cairomm:0
+	dev-cpp/glibmm:2
+	dev-cpp/gtkmm:3.0
+	dev-libs/glib:2
+	dev-libs/jsoncpp:=
+	dev-libs/libinput:=
+	dev-libs/libsigc++:2
+	>=dev-libs/libfmt-7.0.0:=
+	>=dev-libs/spdlog-1.8.5:=
+	dev-libs/date:=
+	dev-libs/wayland
+	gui-libs/wlroots:=
+	x11-libs/gtk+:3[wayland]
+	x11-libs/libxkbcommon
+	evdev? ( dev-libs/libevdev:= )
+	jack? ( virtual/jack )
+	libinput? ( dev-libs/libinput:= )
+	logind? ( || ( sys-apps/systemd sys-auth/elogind ) )
```

Comment:
Could you multi-line this? It's a bit hard to read. 

What does `systemd` use flag do that isn't covered by this? I feel like there's some combining that needs to be done, or if `+systemd` will **just** install service files, then always install those unconditionally.

Reply:
For the systemd USE flag: you are correct, its only purpose is to decide whether to install the systemd unit file. Unfortunately the meson.build file checks for the presence of systemd when this option is enabled, and fetches the unit dir path, so I included a patch that hardcodes the path to `/usr/lib/systemd/user` and removes the check in meson.build.

For the logind USE flag, its purpose is to enable/disable the idle inhibit module, which requires (e)logind

Reply:
Thanks for the effort but let's not overcomplicate things. This patch will make future maintenance harder, unless it is accepted and merged upstream. Then we can obviously add it. 

If it checks that systemd exists, then the use flag makes more sense.

Reply:
@juippis since it's already merged, I'll take a note to remove the patch for the next update (eg. 0.9.18)